tests/example-all-fail/build.sbt

```diff
@@ -1,3 +0,0 @@
-scalaVersion := "2.12.8"
-
-libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % "test"
```

Comment:
I'd argue that this file should still be there (albeit updated to version 2.13.6 that [exercises in the Scala repo use](https://github.com/exercism/scala/blob/main/exercises/practice/all-your-base/build.sbt)), as that better reflects the expected files of an exercise.

Reply:
It is not really documentation, but more like: this is an _actual_ exercise's contents, which matches what the test runner will receive when we run it in production. So it is more about asserting that the test runner works with the actual exercise files.

Reply:
All right, adding those files back :smile:

Reply:
Actually... I see the `build.sbt` file in there, but not the `project/build.properties`. Did I miss something?